### PR TITLE
Make lvgl tick more accurate. (rebase to master)

### PR DESCRIPTION
**Pull request by rockindy**

### CODE REVIEW

File: driver/SDL/modSDL.c [hunk 1]
```diff
@@ -40,9 +40,22 @@ STATIC int tick_thread(void * data)
 {
     (void)data;
 
+    Uint64 pfreq = SDL_GetPerformanceFrequency();
+    Uint64 tick_ms = pfreq / 1000 * LV_TICK_RATE;
```
Comment (amirgon) [inline driver/SDL/modSDL.c:44]: Just for clarity I would add parenthesis.

**Comment by amirgon** [comment]
In general looks good to me.  
What do you think @embeddedt ?

**Comment by embeddedt** [comment]
I am still not convinced about the `* 2` logic. Why can't you just call `lv_tick_inc(acc)` and get rid of the accumulated ticks? LVGL can handle larger-than-normal values being passed to `lv_tick_inc`.

**Comment by rockindy** [comment]
@amirgon , I have added parenthesis, thanks.

@embeddedt , I think avoid using `* 2` logic is great suggestion and has updated new coomit.
`delta - tick_ms` is very small error which is less than 1ms, the `acc` needs several ticks to reach 1ms and more ticks to reach `LV_TICK_RATE`. We should only call `lv_tick_inc()` when the unit of parameter is ms.

**Comment by amirgon** [comment]
@embeddedt - Do you have any more comments about this? Can I merge this?

**Comment by embeddedt** [comment]
Looks good.

**Comment by amirgon** [comment]
Merged.
Thank you!

**Pull request merged**